scripts/update-submodule.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Update python-base submodule and collect release notes.
+# Usage:
+#   scripts/update-submodule.sh
+#
+# Create a commit "generated python-base update" and send a PR to the main repo.
```

Comment:
i don't see it creates a commit for Release notes, and also it does not send a PR, does it?

Reply:
it doesn't. I added a TODO and made the comment clear that the user needs to manually create the commit and PR.